panel/_dataclasses/pydantic.py

```diff
@@ -43,20 +41,15 @@ def observe_field(
 
     @classmethod
     def get_layout(cls, model, self, layout_params):
-        # Todo: Find proper solution to update the layout after the model is updated
-        # Hack: For some reason the model is updated after the UI is rendered
-        # We might setup a callback to update the layout below after the model is updated
-        async def view_model(*args, model=model):
-            if not cls._is_testing:
-                await sleep(0.250)
-            return json.loads(model.json())
+        exclude = list(layout_params)
+        def view_model(*args):
```

Comment:
Why exclude attributes of a Pydantic model?

Reply:
Don't think there was a good reason.